game/scripts/npc/abilities/necrolyte_heartstopper_aura.txt

```diff
@@ -15,7 +15,7 @@
 
     // Casting
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCastRange"                                    "700"
+    "AbilityCastRange"                                    "900"
```

Comment:
maybe set this to 700 and put "oaa, fixing it for valve" behind it, it's a mistake they made, no need to copy it